### ISSUE TITLE
Add. 14,524, foll. 12–21 | https://bl.vuexistapps.us/manuscript/238 (Part5)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1083]
### Problems with Manuscript Overview

- Date: the Roman numerals appear in Upper Case and the word "Century" is not abbreviated as usual. 

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [X] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- Wright mentions that this manuscript has "ornamental devices." So, shouldn't a Decoration section be added? 
- "neatly written" and "of the xith cent." are repeated in the main paragraph of the Phys. Des. and Hand 1. 

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
Resolved:
- Repeated hand info deleted

Question:
Do we add Decoration for this one? I feel like Wright's mention of "ornamental devices" is not really significant for a separate Decoration section.
@wlpotter

**Comment by wlpotter** [comment]
Yes, let's add a decoration note for this one (Wright in general is not very interested in decorations, so most of his descriptions are similarly sparse, but we'd like to surface them so that new scholarship can more easily find decorated Syriac mss). Let me know if you need guidance on adding this element, but I think the guidelines (https://bl.syriac.uk/documentation/guidelines#decodesc) are relatively straightforward here

**Comment by mikey-tang16** [comment]
Decoration section added
